connPFM/tests/test_integration.py

```diff
@@ -60,9 +60,15 @@ def test_integration_debias(
     if skip_integration:
         pytest.skip("Skipping integration test")
     subprocess.call(
-        "connPFM -i {} -a {} --AUC {} -d {} -m {} -tr 1 -u vferrer -nsur 50 -w debias".format(
-            bold_file, atlas_file, AUC_file, surr_dir, ets_auc_denoised_file
-        ),
+        "connPFM -i {} -a {} --AUC {} -d {} -m {} --prefix {} ".format(
+            bold_file,
+            atlas_file,
+            AUC_file,
+            surr_dir,
+            ets_auc_denoised_file,
+            join(testpath, f"{basename(bold_file[:-7])}"),
```

Comment:
Same as before: I think the prefix should not contain the output folder, but rather use the output directory parameter and save the file there.